File: service/src/main/java/com/financial/service/repository/KrBankSchemaRepository.java
package com.financial.service.repository;

import com.financial.service.domain.KrBankSchema;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public
interface KrBankSchemaRepository extends JpaRepository<KrBankSchema, Long> {

}

File: src/main/kotlin/com/minssan9/financial/krbank/repository/krBankSchemaRepository.kt
package com.minssan9.financial.krbank.repository

import com.minssan9.financial.krbank.domain.KrBankData
import com.minssan9.financial.krbank.domain.KrBankSchema
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface krBankSchemaRepository : JpaRepository<KrBankSchema, Long>{

}
